README.md

```diff
@@ -1,7 +1,25 @@
 # UnionPatcher
 
-A tool for LittleBigPlanet that patches the server urls to a custom url.
+A commandline tool that will take the official EBOOT of a LittleBIGPlanet title and replace the server URLs with the ones specified
 
-# Downloading
-You can download UnionPatcher by going [here](https://github.com/LBPUnion/UnionPatcher/actions), clicking the top link, and choosing a download for your platform at the bottom.
-Mac will have to be self-compiled for now, but it does work.
+## Prerequisites
+* For running the console application, you will need the [.NET 6 Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/6.0/runtime?utm_source=getdotnetcore&utm_medium=referral)
+* You will need to know the server URL you wish to use
+* Basic commandline knowledge
```

Comment:
```suggestion
```

Reply:
Removed the references to the commandline